### DISCUSSION TITLE
How to define an oritation in Mtex

### DISCUSSION THREAD

**Discussion by maguag** [mtex-toolbox/mtex#1654] [Ask Anything] [answered]
Dear Developers,

I am trying to plot some orientation figures just like follows, so how can i define an orientation just like (110)<001>, and then choose the grains which are similar to this orientation, and define the grains' color ? Now I can only choose a grain and then plot.
Thanks.

```matlab
[grains,ebsd('Iron bcc').grainId] = calcGrains(ebsd('Iron bcc'));
figure();
plot(grains.boundary,'lineWidth',2,'micronbar','off');
hold on
plot(grains(13),'FaceColor','yellow');
hold off
hold on
grains_selected1 = grains.findByOrientation(grains(13).meanOrientation,15*degree)
plot(grains_selected1,'FaceColor','yellow');
hold off
hold on
grains = calcGrains(ebsd('Iron bcc'),'angle',1*degree);
grains = smooth(grains,2);
plot(grains.boundary,'lineWidth',2);
hold off
```
![8f5f1e100df81f17bf8719d9b82c101](https://user-images.githubusercontent.com/48279761/201874354-b76526bd-29da-4e5e-a77a-7ea6be7ce55e.jpg)

**Comment by maguag** [comment]
Now I can only select grains by id, it's different from analysed by AZTec softwere. 
![11](https://user-images.githubusercontent.com/48279761/201878327-d40c5a66-f78f-4f45-808f-ca7545a16500.jpg)

**Comment by ralfHielscher** [comment, marked as answer]
Maybe you are looking for something like this:
```matlab
ori = orientation.byMiller([1 1 0],[0 0 1],ebsd('Iron bcc').CS);
grains_selected1 = grains.findByOrientation(ori,15*degree)
```
Ralf.

**Reply by maguag** [reply]
Thank you, it really helps.

**Reply by Brian-Liu-rx** [reply]
Hi,  the findByOrientation here is for the mean orientation of grain, right?

How about for the pixels?

Thanks!
